src/ListGroupItem.js

```diff
@@ -1,97 +1,43 @@
-import React, { cloneElement } from 'react';
```

Comment:
Probably easier to simply view this file than look at the diff, its now just a simple render function.